src/pages/item/item.spec.ts

```diff
@@ -0,0 +1,115 @@
+import { ComponentFixture, async, fakeAsync, tick } from '@angular/core/testing';
+import { NgForm } from '@angular/forms';
+import { TestUtils } from '../../test';
+import { ItemPage } from './item';
+import { InventoryDataMock } from '../../mocks';
+
+let fixture: ComponentFixture<ItemPage> = null;
+let instance: any = null;
+
+describe('Item Page', () => {
+
+  beforeEach(async(() => TestUtils.beforeEachCompiler([ItemPage]).then(compiled => {
+    fixture = compiled.fixture;
+    instance = compiled.instance;
+  })));
+
+  afterEach(() => {
+    fixture.destroy();
+  });
+
+  it('is created', () => {
+    expect(instance).toBeTruthy();
+    expect(fixture).toBeTruthy();
+  });
+
+  it('gets navParam tag', () => {
+    instance.navParams.param = 'banana';
+    instance.ngOnInit();
+    expect(instance.tag).toEqual('banana');
+  });
+
+  it('gets navParam action', () => {
+    instance.navParams.param = 'Edit';
+    instance.ngOnInit();
+    expect(instance.tag).toEqual('Edit');
+  });
+
+  it('initializes with \'Good\' as condition default value', () => {
+    instance.ngOnInit();
+    expect(instance.item.condition).toEqual('Good');
+  });
+
+  it('gets item if action === \'Edit\'', fakeAsync(() => {
+    let item = { brand: 'Canon', model: 'Rebel T5i', category: 'Camera', cost: '750', condition: 'Good' };
```

Comment:
There is a lot of duplication of test data here and below. In the API, I created a [fixtures](https://github.com/AdamVig/stockpile-api/blob/master/test/fixtures/db.json) file that I then used for [the db service test](https://github.com/AdamVig/stockpile-api/blob/master/test/db.js). Helped to reduce duplication and to allow for separation of data from logic.

Do you think it would make sense to do something similar here?